gcloud-java-gax/src/main/java/io/gapi/gax/grpc/ServiceApiSettings.java

```diff
@@ -81,19 +85,19 @@ public Credentials getCredentials() {
   }
 
   /**
-   * The path used to reach the service.
+   * The path used to reach the service. This value will not be used if the channel is set.
    */
-  public ServiceApiSettings setServicePath(String servicePath) {
-    this.servicePath = servicePath;
+  public ServiceApiSettings setServiceAddress(String serviceAddress) {
+    this.serviceAddress = serviceAddress;
     return this;
   }
 
-  public String getServicePath() {
-    return servicePath;
+  public String getServiceAddress() {
```

Comment:
For the record (we did discuss it), my preference would be either to call this method `getAddress` (as part of `ServiceApiSettings` I think it should be clear that this is a service address) or, less prefered, change `getPort` to `getServicePort` for consistency.